database/sql/V029__create_onchain_placed_orders.sql

```diff
@@ -0,0 +1,20 @@
+-- Table to store all orders placed via any on-chain broadcaster contract
+-- uid is needed as unique identifier for the order
+-- Sender is the address who called the broadcasting contract. For ethflow orders, this would be the user placing the order
+-- is_reorged is a flag indicated whether the order was placed and then reorged.
+-- block_number + log_index are used to deal with chain reverts.
+-- All other information about the order is directly translated into an order and the information is stored in the orders table
```

Comment:
Good question. While signatures do not need to be valid for EIP1271 orders, the question is legitimate, as there can be many reasons for an invalid order: E.g if the valid_to is in the past, the orders would also not be valid.

Since we won't refund invalid orders, I think we don't have to store them right now. But yeah, in a future version, we could add a field that allows storing the reason for the invalidity of the order. But since this is an "additional feature", and it can be coded up later, I think we will not have it in the first version.